app/src/main/kotlin/com/simplemobiletools/calculator/activities/MainActivity.kt

```diff
@@ -11,19 +10,18 @@ import com.simplemobiletools.calculator.BuildConfig
 import com.simplemobiletools.calculator.R
 import com.simpletools.calculator.commons.extensions.config
 import com.simpletools.calculator.commons.extensions.updateViewColors
-import com.simplemobiletools.commons.extensions.*
 import com.simplemobiletools.commons.helpers.LICENSE_AUTOFITTEXTVIEW
 import com.simplemobiletools.commons.helpers.LICENSE_ESPRESSO
 import com.simplemobiletools.commons.helpers.LICENSE_KOTLIN
 import com.simplemobiletools.commons.helpers.LICENSE_ROBOLECTRIC
 import com.simpletools.calculator.commons.activities.SimpleActivity
-import com.simpletools.calculator.commons.helpers.*
-import kotlinx.android.synthetic.main.activity_main.*
 import me.grantland.widget.AutofitHelper
-import android.net.NetworkInfo
-import android.content.Context.CONNECTIVITY_SERVICE
-import android.net.ConnectivityManager
 
+/* ktlint-disable no-wildcard-imports */
```

Comment:
Why not use only the specific imports?

Reply:
I can fix it but it will require a lot of imports